test/parallel/forall/reduce-intents/ri-bool-baseline-bug.chpl

```diff
@@ -0,0 +1,8 @@
+var A: [false..true] real = [1.2, 3.4];
+
+var locIdx = max(bool);
+forall i in A.domain with (min reduce locIdx) {
+  if (i < locIdx) {
+  }
```

Comment:
did you want to set `locIdx` here?

Reply:
It doesn't need to be to hit the internal error, and I was trying to keep the reproducer as minimal as possible.

Reply:
sounds good